process-dir.js

```diff
@@ -164,16 +166,30 @@ async function replacePackage (dir, file, files) {
 }
 
 async function process (dir) {
+  let ignorePatterns = []
+  let removePatterns = []
+
   let npmignorePath = join(dir, '.npmignore')
-  let ignore = []
   if (fs.existsSync(npmignorePath)) {
-    ignore = await readFile(npmignorePath)
-    ignore = ignore.toString().split('\n').filter(i => !!i).map(i => {
-      return i.endsWith(sep) ? i.slice(0, -1) : i
-    })
+    removePatterns =
```

Comment:
Can you revert it? I prefer the old code style.

Reply:
Ok. I changed the code style because the string was more than 80 characters length